source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,167 +1,209 @@
 
-The first method is to use the :ref:`mixer tool<mixer>` to create your own
-|CL| image and add your bundles to it.  Mixing your own |CL| image can
-give you great control and flexibility; however, you must act as an
-:abbr:`OSV (Operating System Vendor)` and maintain your releases and
-updates because you have forked from upstream.
+If you need to customize beyond what the mixin tool provides, use the
+:ref:`mixer tool<mixer>`.
 
-The second method is to use the :command:`mixin` tool, which also
-makes use of mixer to create custom bundles that you can add to your
-upstream |CL| system.  This  simpler method provides a “light” forking from
-upstream, which means you can continue to get upstream bundles and updates.
-If needed, you can easily revert your system back to the upstream version.
+Prerequisites
+*************
 
-This guide shows you how to accomplish the second method by following these
-steps:
+To start working with the mixin tool, you will need a recent image of |CL| with
+the `mixer` bundle installed. Before you install any new packages, update your
+OS with the following command:
 
-#. Set up the workspace.
-#. Copy your custom RPM package to the workspace.
-#. Create a bundle with your custom RPM package.
-#. Migrate your |CL| system to your custom mix.
-#. Add your custom bundle to your system.
-#. Optional: Revert your system back to 100% upstream.
+.. code-block:: bash
 
-Set up the workspace
-********************
+   sudo swupd update
 
-#. Install the mixer bundle to enable mixer.
+If the bundle is not yet installed, you can add it with the :command:`swupd bundle-add`
+command as follows:
 
-   .. code-block:: console
+.. code-block:: bash
 
-      $ sudo swupd bundle-add mixer
+   sudo swupd bundle-add mixer
 
-#. Create the workspace.
+Mixin workflow
+**************
 
-   .. code-block:: console
+The workflow to create your own mixin is outlined below.
 
-      $ sudo mkdir -p /usr/share/mix/local-rpms
+#. :ref:`create-workspace-mixin`
+#. :ref:`create-rpm`
+#. :ref:`copy-custom-rpm`
+#. :ref:`create-bundle`
+#. :ref:`migrate-mix`
+#. :ref:`add-bundle-to-system`
+#. :ref:`revert-to-upstream`
 
-Copy your custom RPM package to the workspace
-*********************************************
+.. _create-workspace-mixin:
 
-.. note::
+Create a workspace
+******************
 
-   You cannot simply use RPMs from other Linux distros on |CL|. You must
-   build RPMs specifically for |CL| in order for them to work properly.
-   Follow the instructions on how to build RPMs found at the
-   `Developer tooling framework for Clear Linux`_.  
+Use the following command to create an empty directory in your |CL| image to
+use as a workspace for mixing:
 
-If you have a local RPM you want to add to your mix you can do so by copying
-your RPM package to the workspace.
+.. code-block:: bash
 
-.. code-block:: console
+   sudo mkdir -p /usr/share/mix/local-rpms
 
-   $ sudo cp [RPM] /usr/share/mix/local-rpms
+.. _create-rpm:
```

Comment:
The next step should be add custom RPMs to your mixin repo if you have your own custom RPMs.  It's not "create RPM".